src/avatar.scss

```diff
@@ -64,7 +64,13 @@ $block: #{$fd-namespace}-avatar;
   }
 
   &__icon {
-    @include fd-icon-element-base();
+    @include fd-icon-element-base() {
+      // Added in order to trully center the icon inside it's container
+      display: flex;
+      align-content: center;
+      justify-content: center;
+      align-items: center;
+    }
```

Comment:
The `_mixins.scss` file contains reusable mixins such as `fd-flex-center`. 
<img width="348" alt="Screen Shot 2021-04-28 at 11 41 51 AM" src="https://user-images.githubusercontent.com/39598672/116432783-e0407680-a816-11eb-906d-6bca7af1e5bc.png">

You can call here the mixin like this:
```
@include fd-flex-center() {
        align-content: center;
}
```

Reply:
You can remove the comment on line 68

Reply:
Done. Please re-check